### DISCUSSION TITLE
Default language for template tag

### DISCUSSION THREAD

**Discussion by rafasoares** [nuxtrdev/nuxtr-vscode#33] [Ideas]
We can select the default `lang` for `<script>` and `<style>` tags, but not for `<template>`. Maybe that was an oversight?

I use Pug as a template engine, it would be lovely if components were already created with that setup (and with an empty `<template>`, instead of having the default `<div></div>` - which is invalid in Pug). I know I could use default templates for this, but it seems overkill, not to mention a bit annoying to do for every project.

Otherwise, I'm loving the extension and it's been a great time saver 😄

**Comment by adhamfarrag** [comment]
Hey Rafael @rafasoares 

Thanks for your kind words and support, definitely it's an oversight. 😅

I'll add it to the next release but a workaround should be making your own default page / layout templates without <div> and Nuxtr will use them for future Vue files except for Components. 

Here's how to do it [Vue File Templates](https://github.com/nuxtrdev/nuxtr-vscode#vue-file-templates).

**Reply by adhamfarrag** [reply]
Hey @rafasoares. Two questions, 

1. Do you personally use linters with Pug? If so, what do you use? 

2. Can we have a quick chat with you about integrating Pug? Having couple questions. :) 

Thanks.

**Reply by rafasoares** [reply]
Hey, @adhamfarrag! I'm happy to give you answers and help out however I can! 😄 

1. Yep, I was using ESLint with StandardJS and the official Vue plugin in the past and it did lint things like max attributes per line, order of attributes (specifically: `boolean` attrs, then `normal="attrs"`, then `:bound`, then `@event`), etc. Nowadays I'm using Prettier, but I still haven't gotten around to properly setting up to lint/format `.vue` files - let alone Pug.
2. Sure thing! Let me know how do you want to it.

**Reply by adhamfarrag** [reply]
@rafasoares Hey, somehow I missed the notification for your message. Do you use Discord ? My username is the same `adhamfarrag` or just share yours :)
